app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    def __init__(self, name, appetite, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+    
+    def print_name(self):
+        print(f"Hello, I'm {self.name}.")
+    
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points . . .")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+class Cat(Animal):
+    def __init__(self, name, is_hungry=True):
+        self.name = name
+        self.is_hungry = is_hungry
```

Comment:
You don't need to set `self.name` and `self.is_hungry` here, as these are already being set in the superclass `__init__` method. You can remove these lines.